cmd/cli/main.go

```diff
@@ -53,6 +54,16 @@ func get_action() types.Action {
 
 func main() {
 	flag.Parse()
+
+	switch *flag_operation {
+	case "generate":
+		generatePayload()
+	case "query":
+		query.QueryProof(*flag_platform, *flag_identity)
+	}
```

Comment:
Always add a `default:` and outputs an error message to let users know if they inputed something wrong.